OneCommunityGlobal/HighestGoodNetworkApp
# Ramya teams limit see all

## Note: 
Taking a look at the recent reviews. Kindly do not review the PR unless you see a different kind of bug to report

# Description of the feature

- A “Limit See-All” option for teams 

-  Other Links ->Teams

- When this option is turned on for a team member,  the team member  shouldn't  be able to see  other members of the team in their dashboard 

- The member however should be able to see team members of other teams  they are part of 



## Related PRS (if any):
This frontend PR is related to the [731](https://github.com/OneCommunityGlobal/HGNRest/pull/731) backend PR.

## Main changes explained:
-  Added a column ‘See all’ in the "Team Member " popup
-  A tool tip beside See All
-  Added a Toggle Switch in Add/Edit Teams pop-up beside every row 
-  Rendered the toggle switch with appropriate choice.
-  When the user makes the choice,  make a call to   action function 
-  The  action function  makes a post call to the backend 
-  Make a dispatch call to the reducer to update the redux store. 

## How to test:
1.  check into current branch
2.  do `npm install` and `...` to run this PR locally
3.  log as admin user
4.  Go to Other Links ->Teams
5.  Create two Teams , lets say Team A and Team B (you can use any existing Team as well)
6.  Add  the a **_testuser_1_** as team member to **_Team A_** **_and Team B_**
7.  Set the toggle switch to off  in Team A for **_testuser_1_**
8.  Login to the **_testuser_1_**'s  profile and verify if the user is only able  to see  the members of the  Team B and not Team A

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/89441435/2eaaf5a2-9f80-442c-9ace-20fddc997f35


## Note:

- Kindly mention any  bugs , errors you see in the console. 
- Try breaking the functionality , any suggestion is helpful
